_wiki/FAQ/General.md

```diff
@@ -2,63 +2,112 @@
 title: General FAQ
 ---
 
-### Application Data Directory {#appdata}
+### Application Data Directory
 
 | Operating System | Path |
-| :--------------: | ---- |
-| Windows          | `%localappdata%\OpenTabletDriver`                  |
-| Linux            | `~/.config/OpenTabletDriver`                       |
-| macOS            | `~/Library/Application Support/OpenTabletDriver`   |
+| --- | --- |
+| Windows | `%localappdata%\OpenTabletDriver` |
+| Linux | `~/.config/OpenTabletDriver` |
+| macOS | `~/Library/Application Support/OpenTabletDriver` |
 
----
-
-### Command Line Arguments {#command-line-args}
+### Command Line Arguments
 
 #### UI
 
 | Argument | Description |
-| -------- | ----------- |
-| `--minimized` or `-m` | Starts the application in a minimized state. |
+| --- | --- |
+| `--minimized` or `-m` | Starts the application in a minimized state |
 
 #### Daemon
 
 | Argument | Description |
-| -------- | ----------- |
-| `--appdata` or `-a` | Specifies the application data directory. |
-| `--config` or `-c`  | Specifies the configurations directory. |
+| --- | --- |
+| `--appdata` or `-a` | Sets the application data directory. |
```

Comment:
I don't think this is necessarily better wording. "Sets" sounds a little bit like it permanently sets the setting. How about "Use the specified application data directory"
(same goes for other `Specifies -> Sets` replacements)